### ISSUE TITLE
feat[design]: ✨ showcase page - concept

### ISSUE THREAD

**Issue by coraliefeil** [solid-design-system/solid#1145]
## Description
This tasks describes the Figma implementation for: #1147 

- [x] concept
- [x] research - benchmark
- [x] landing page design
- [x] present to SDS team
- [x] mobile design
- [x] prepare visuals

Figma Link[
](https://www.figma.com/design/17uCzMkjB59TL0npQmjXE0/Showcase-Page?node-id=1-51&t=Qz45dLTRrLyTDNYk-11)

## DoR
- [x] Item has been estimated by the team
- [x] Item dependencies have been identified and documented

## DoD
- [ ] Brand approval (if neccessary)
- [ ] Figma documentation file incl. hand-off has been created/updated (if applicable)
- [ ] Tokens have been applied/updated (if applicable)
- [ ] Crosslinks are created/updated
- [ ] Implementation works successfully on `figma` branch

**Comment by van-nguyen-ht** [comment]
@coraliefeil I've created the one pager [here](https://www.figma.com/design/17uCzMkjB59TL0npQmjXE0/Showcase-Page?node-id=0-1&t=jfo6aHnFEDTLdBRy-11)

**Comment by coraliefeil** [comment]
@MartaPintoTeixeira you could try out some layout options for the showcase page until my return. @miguelxnxn can brief you in.

Pls collaborate and find out what can be communicated in the content part of the page and how this can look like.
Any ideas welcome. Have fun.

**Comment by MartaPintoTeixeira** [comment]
I had a meeting with Miguel yesterday and he has briefed me on the showcase page project.
I have some proposals already and will discuss the with him.

**Comment by MartaPintoTeixeira** [comment]
Hi @yoezlem I just had a quick call with Miguel and he said that would be helpfull if you could have a look at the whom section in the Showcase figma file (I have just commented and tagged you on the section).

![Image](https://github.com/user-attachments/assets/4d8496d3-6596-407a-9199-780beec39eff)

If you could work on the copy and if possible simplify the personas it would be helpfull.

**Comment by yoezlem** [comment]
@MartaPintoTeixeira added some comments to the file.

**Comment by coraliefeil** [comment]
Left some comments in figma

**Comment by MartaPintoTeixeira** [comment]
@coraliefeil please check the mobile version for the Showcase page

**Comment by MartaPintoTeixeira** [comment]
Animation Video Quick Previews:
https://github.com/user-attachments/assets/c5571685-3a18-4f2f-9759-e000c6e2bbc1Previews
https://github.com/user-attachments/assets/d78d7fe6-a530-4e79-ba1a-99640e599572
https://github.com/user-attachments/assets/6ba67d55-1453-491c-a84c-b6ed7c93b22d
https://github.com/user-attachments/assets/f20d2768-c3dc-4ae0-8f5a-d79873d9b1a5
https://github.com/user-attachments/assets/585bf63f-f20a-4aa9-b58c-39530f3df80e
https://github.com/user-attachments/assets/db13afbb-61a0-4e30-8602-5f13d4229c18

**Comment by coraliefeil** [comment]
Yeah, we can close showcase page and open up a new issue that handles accompanying code implementation and asset refinement.
